Sources/MongoDBVapor/MongoDBVapor.swift

```diff
@@ -1 +1,49 @@
-private func doSomething() {}
+import MongoSwift
+import Vapor
+
+extension Application {
+    /// A global `MongoClient` for use throughout the application. The client is thread-safe
+    /// and backed by a pool of connections so it should be shared across event loops.
+    public var mongoClient: MongoClient {
+        get {
+            self.storage[MongoClientKey.self]!
+        }
+        set {
+            self.storage[MongoClientKey.self] = newValue
+        }
+    }
+
+    private struct MongoClientKey: StorageKey {
+        // swiftlint:disable nesting
+        typealias Value = MongoClient
+    }
+
+    /**
+     * Configures a global `MongoClient` with the specified options. The client will use the application's
+     * `EventLoopGroup` for executing operations.
```

Comment:
can we add that that the client may be accessed via the `mongoClient` computed property on the `Application`? Also perhaps a code sample would be nice. Check [here](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/CodeBlocks.html#//apple_ref/doc/uid/TP40016497-CH12-SW1) for the syntax / format. I think four backticks (\`\`\`\`) seems preferable to the indentation based format.